payments-core/src/main/java/com/stripe/android/payments/core/authentication/WebIntentAuthenticator.kt

```diff
@@ -100,7 +105,7 @@ internal class WebIntentAuthenticator(
         shouldCancelIntentOnUserNavigation: Boolean = true
     ) = withContext(uiContext) {
         val paymentBrowserWebStarter = paymentBrowserAuthStarterFactory(host)
-        logger.debug("PaymentBrowserAuthStarter#start()")
+        Logger.getInstance(enableLogging).debug("PaymentBrowserAuthStarter#start()")
```

Comment:
Wouldn't it be better to continue to inject the logger?

Reply:
This is the only place in the entire graph that the instance is used. Given that the `getInstance` call is just a query for noop or real singleton static logger instance, thought it is cleaner to just use the `enableLogging` boolean to query it when needed